### ISSUE TITLE
Lien absent vers le pack 3.9.2 vers 3.9.3

### ISSUE THREAD

**Issue by fontanil** [AFUJ/joomla.fr#80]
Bonjour,
Sur la page de téléchargement des mises à jour 3.9.3, le lien 3.9.2 vers 3.9.3 est celui de la page d'accueil et non celui du pack.
Cordialement,
Robert

**Comment by webmastercivis** [comment]
https://www.joomla.fr/joomla/telecharger-joomla, idem pour moi, lien 3.9.2 vers 3.9.3 renvoi à l'accueil

**Comment by infograf768** [comment]
Il semble que ceci ait été corrigé

**Comment by fontanil** [comment]
Pas en ce qui me concerne. Portail Joomla.fr, Joomla! en français, 3.9.2 vers 3.9.3 renvoie toujours vers l’accueil du portail.

 

De : infograf768 <notifications@github.com> 
Envoyé : vendredi 8 mars 2019 11:36
À : AFUJ/joomla.fr <joomla.fr@noreply.github.com>
Cc : fontanil <ergemel@gmail.com>; Author <author@noreply.github.com>
Objet : Re: [AFUJ/joomla.fr] Lien absent vers le pack 3.9.2 vers 3.9.3 (#80)

 

Il semble que ceci ait été corrigé

**Comment by sebastienlapoux** [comment]
Bonjour,

Bien corrigé comme dit par @infograf768 . Supprimer le cache de votre navigateur si besoin.
![image](https://user-images.githubusercontent.com/13470265/54032085-e72c4b00-41b0-11e9-83a9-07ec83d1d263.png)

Si besoin le lien https://www.joomla.fr/joomla/telecharger-joomla?task=download&collection=seb_download_full_package&xi=0&file=seb_download_full_package&id=12867

Bien cordialement.

**Comment by fontanil** [comment]
Désolé, mais cache vidé, cookies supprimés, pas de changement ! 

Changement de navigateur (4 différents : Firefox, Chrome, Opera et Edge) : le lien est toujours celui de l’accueil du portail.

 

De : Sebastien Lapoux <notifications@github.com> 
Envoyé : vendredi 8 mars 2019 14:47
À : AFUJ/joomla.fr <joomla.fr@noreply.github.com>
Cc : fontanil <ergemel@gmail.com>; Author <author@noreply.github.com>
Objet : Re: [AFUJ/joomla.fr] Lien absent vers le pack 3.9.2 vers 3.9.3 (#80)

 

Bonjour,

Bien corrigé comme dit par @infograf768 <https://github.com/infograf768>  . Supprimer le cache de votre navigateur si besoin.
 <https://user-images.githubusercontent.com/13470265/54032085-e72c4b00-41b0-11e9-83a9-07ec83d1d263.png> 

Si besoin le lien https://www.joomla.fr/joomla/telecharger-joomla?task=download <https://www.joomla.fr/joomla/telecharger-joomla?task=download&collection=seb_download_full_package&xi=0&file=seb_download_full_package&id=12867> &collection=seb_download_full_package&xi=0&file=seb_download_full_package&id=12867

Bien cordialement.

**Comment by com3elles** [comment]
Bonjour,
voyant ces échanges passer, j'ai testé à mon tour et je confirme que le lien n'est pas bon. Cache vidé idem.
![image](https://user-images.githubusercontent.com/4996091/54039208-a9382280-41c2-11e9-98d3-97a86c9ef35e.png)

**Comment by fontanil** [comment]
Oui, il faudrait peut-être lire ce qui a été écrit encore tout à l'heure avant d'affirmer le contraire !
"Portail Joomla.fr, Joomla! en français, 3.9.2 vers 3.9.3 renvoie toujours vers l’accueil du portail." 
Je n'ai jamais parlé du pack complet... :-(
